icefall/utils.py

```diff
@@ -529,25 +529,42 @@ def __mul__(self, alpha: float) -> "MetricsTracker":
         return ans
 
     def __str__(self) -> str:
-        ans = ""
+        ans_frames = ""
+        ans_utterances = ""
         for k, v in self.norm_items():
             norm_value = "%.4g" % v
-            ans += str(k) + "=" + str(norm_value) + ", "
+            if "utt_" not in k:
+                ans_frames += str(k) + "=" + str(norm_value) + ", "
+            else:
+                ans_utterances += str(k) + "=" + str(norm_value)
+                if k == "utt_duration":
+                    ans_utterances += " frames, "
+                else:
```

Comment:
I suggest using
```python3
elif k == "utt_pad_proportion":
  ans_utterances += ", "
else:
  raise ValueError(f"Unexpected key: {k}")
```

If we later add some more options with `utt_xxx`, it can
save us some time to find the right place to add code to support it.